File: clients/go/my_strategy.go
package main

import (
    . "komus24/model"
    . "komus24/debugging_interface"
)

type MyStrategy struct{}

func NewMyStrategy(constants Constants) *MyStrategy {
    return &MyStrategy{}
}

func (strategy MyStrategy) getOrder(playerView PlayerView, debugInterface *DebugInterface) Order {
    return Order {
               Vehicles: make([]VehicleOrder, 0),
           }
}

func (strategy MyStrategy) debugUpdate(displayedTick int32, debugInterface DebugInterface) {}

func (strategy MyStrategy) finish() {}

File: clients/csharp/MyStrategy.cs
namespace Komus24
{
    public class MyStrategy
    {
        public MyStrategy(Komus24.Model.Constants constants) {}
        public Komus24.Model.Order GetOrder(Komus24.Model.PlayerView playerView, DebugInterface debugInterface)
        {
            return new Komus24.Model.Order(new Komus24.Model.VehicleOrder[0]);
        }
        public void DebugUpdate(int displayedTick, DebugInterface debugInterface) {}
        public void Finish() {}
    }
}